### PR TITLE
project init

### PR DESCRIPTION
**Pull request by Andrei-Suta**

### CODE REVIEW

File: src/app.module.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Module } from "@nestjs/common";
+import { ConfigModule, ConfigService } from "@nestjs/config";
+import { TypeOrmModule } from "@nestjs/typeorm";
+import { AppController } from "./app.controller";
+import { AppService } from "./app.service";
```
Comment (danielursu) [inline src/app.module.ts]:
```suggestion
import { TypeOrmModule } from "@nestjs/typeorm";

import { AppController } from "./app.controller";
import { AppService } from "./app.service";
```

File: src/main.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { NestFactory } from "@nestjs/core";
+import { AppModule } from "./app.module";
```
Comment (danielursu) [inline src/main.ts]:
```suggestion
import { NestFactory } from "@nestjs/core";

import { AppModule } from "./app.module";
```

File: src/main.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+async function bootstrap() {
```
Comment (danielursu) [inline src/main.ts]:
```suggestion
async function bootstrap() {
```
see what app.listen() is returning

File: test/app.e2e-spec.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import * as request from 'supertest';
+import { AppModule } from './../src/app.module';
```
Comment (danielursu) [inline test/app.e2e-spec.ts]: double quotes

File: .env [hunk 5]
```diff
@@ -0,0 +1,5 @@
+POSTGRES_HOST='localhost'
```
Comment (danielursu) [inline .env]: double quotes

**Review by danielursu — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (danielursu) [inline .gitignore]: just node_modules?

**Review by danielursu — CHANGES_REQUESTED** [review]

**Review by mgandore — APPROVED** [review]

File: .gitignore [hunk 7]
```diff
@@ -1 +1,61 @@
-node_modules
+# Created by https://www.toptal.com/developers/gitignore/api/angular,visualstudiocode
+# Edit at https://www.toptal.com/developers/gitignore?templates=angular,visualstudiocode
```
Comment (danielursu) [inline .gitignore]: this is gitingore from angular deps
try this instead https://github.com/nestjs/nest/blob/master/.gitignore

**Review by danielursu — CHANGES_REQUESTED** [review]

**Review by danielursu — APPROVED** [review]

**Pull request merged**